pkg/schema/generate.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"github.com/alecthomas/jsonschema"
+	"io/ioutil"
+	"k8s.io/client-go/tools/clientcmd/api"
+	"sigs.k8s.io/yaml"
+	"strings"
+)
+
+const outpufFile = "site/content/usage/20-schema.md"
+
+func main() {
+
+	var document strings.Builder
+	document.WriteString("---\n")
+	document.WriteString("title: Config file schema\n")
+	document.WriteString("weight: 200\n")
+	document.WriteString("---\n\n")
+	document.WriteString("```yaml\n")
+
+	schema := jsonschema.Reflect(&api.Config{})
+	yamlSchema, _ := yaml.Marshal(schema.Definitions)
```

Comment:
Should we ignore the error? Looks like `errcheck` warns if the error isn't checked.